scripts/Tools/check_input_data

```diff
@@ -62,91 +57,6 @@ formatter_class=argparse.ArgumentDefaultsHelpFormatter
-        else:
-            return True
-
-###############################################################################
-def check_input_data(svn_loc, input_data_root, data_list_dir, download):
-###############################################################################
-    """
-    Return True if no files missing
-    """
-    expect(os.path.isdir(input_data_root), "Invalid input_data_root directory: '%s'" % input_data_root)
-    expect(os.path.isdir(data_list_dir), "Invalid data_list_dir directory: '%s'" % data_list_dir)
-
-    data_list_files = find_files(data_list_dir, "*.input_data_list")
-    expect(data_list_files, "No .input_data_list files found in dir '%s'" % data_list_dir)
-
-    case = Case()
-    no_files_missing = True
-
-    for data_list_file in data_list_files:
-        logging.info("Loading input file: '%s'" % data_list_file)
-        with open(data_list_file, "r") as fd:
-            lines = fd.readlines()
-
-        for line in lines:
-            line = line.strip()
-            if (line and not line.startswith("#")):
-                tokens = line.split('=')
-                description, full_path = tokens[0].strip(), tokens[1].strip()
-                if(full_path):
-                # expand xml variables
-                    full_path = case.get_resolved_value(full_path)
-                    rel_path  = full_path.replace(input_data_root, "")
-
-                    if (not os.path.exists(full_path)):
-                        model = os.path.basename(data_list_file).split('.')[0]
-                        logging.warning("Model %s missing file %s = '%s'" % (model,description,full_path))
-
-                        if (download):
-                            success = download_if_in_repo(svn_loc, input_data_root, rel_path)
-                            if (not success):
-                                # If ACME, try CESM repo as backup
-                                if (get_model() == "acme" and svn_loc != SVN_LOCS["cesm"]):
-                                    success = download_if_in_repo(SVN_LOCS["cesm"], input_data_root, rel_path)
-                                    if (not success):
-                                        no_files_missing = False
-                                else:
-                                    no_files_missing = False
-                            else:
-                                no_files_missing = False
-                        else:
-                            logging.info("Already had input file: '%s'" % full_path)
-                else:
-                    model = os.path.basename(data_list_file).split('.')[0]
-                    logging.warning("Model %s no file specified for %s"%(model,description))
-
-    return no_files_missing
-
-###############################################################################
 def _main_func(description):
 ###############################################################################
     if ("--test" in sys.argv):
```

Comment:
For some reason github won't let me annotate line 68.   But the arguments to check_input_data are wrong here - should have a case first.